handbook/product/README.md

```diff
@@ -77,7 +77,9 @@ For external contributors: please consider opening an issue with reference scree
 
 Once the draft has been approved, it moves to the "Settled" column on the drafting board. 
 
-Before assigning an engineering manager for [estimation](https://fleetdm.com/handbook/engineering#sprint-ceremonies), the product team should ensure the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
+Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product team should ensure the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
```

Comment:
```suggestion
Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product designer ensures the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
```

Reply:
cc @ireedy

Reply:
Always trying to reduce the surface area of the handbook if we can. We might not be able to here.

Reply:
@noahtalerman good point. I think we can remove the callout box now that we've specified the DRI within the section text. I'll update that now.